controllers/users.js

```diff
@@ -30,6 +35,33 @@ const getCurrentUser = async (_id) => {
   return currentUser;
 }
 
+const verificationUser = async (verificationToken) => {
+  const user = await User.findOneAndUpdate(
+    { verificationToken },
+    { verify: true, verificationToken: null }
+  );
+  return user;
+};
+
+const sendVerificationEmail = async (verificationToken) => {
+  sgMail.setApiKey(process.env.SENDGRID_API_KEY)
+  const msg = {
+    to: 'example@example.pl',
```

Comment:
Mail powinien być wysyłany do podanego maila podczas rejestracji

Reply:
Faktycznie przeoczyłem to :P poprawione :D